src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter, Route } from "react-router-dom";
+import Home from './Pages/HomePage.js';
+import TripsPage from './Pages/TripsPage.js';
+import Trips from '../src/Components/Trips.js';
+
+
```

Comment:
remove extra line and use consistent quotes throughout the app either "  or ', use only one